smart_contracts.rst

```diff
@@ -208,12 +208,12 @@ Fields
 |  partner_signature     | bytes      | Elliptic Curve 256k1 signature of the partner on the withdraw data              |
 +------------------------+------------+---------------------------------------------------------------------------------+
 
-.. _cooperative-settle-proof-message:
+.. _cooperative-settle-proof:
 
 Cooperative Settle Proof
 ------------------------
 
-Data required by the smart contracts to allow the two channel participants to close and settle the channel instantly, in one transaction. It contains the cooperative settle message data and signatures from both participants on the cooperative settle message.
+Data required by the smart contracts to allow the two channel participants to close and settle the channel instantly, in one transaction. It contains the cooperative settle information and signatures from both participants on the cooperative settle information.
```

Comment:
I would change `information` to `data` here. The "signature on the information" part would sound better as "signature on the data"

Reply:
I got close.  I wrote `data` first and replaced `data` with `information` :).  Going back to `data`.

Reply:
Wait, we are talking about "cooperative settle proof" here obviously.

Reply:
Settled with "cooperative settle proof" and refactored the sentence a bit.

Reply:
looks good.